src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -68,7 +68,7 @@ static u8 sDoorAnimOpenFrames[] = { 25, 25, 25, 25 };
 static u8 sDoorAnimCloseFrames[] = { 60, 70, 60, 70 };
 
 static Gfx* D_809FCEE4[5][2] = {
-    { gDoorLeftDL, gDoorRightDL }, { gFireTempleBombableDoorDL2, gFireTempleBombableDoorDL1 },
+    { gDoorLeftDL, gDoorRightDL }, { gFireTempleDoorWithHandleFrontDL, gFireTempleDoorWithHandleBackDL },
```

Comment:
This could probably be simplified by renaming gFireTempleDoorFrontDL,gFireTempleDoorBackDL to gFireTempleShutterDoorFrontDL, gFireTempleShutterDoorBackDL, and then just calling these DoorFront/DoorBack